### PR TITLE
Run pg_format on sql files

### PR DESCRIPTION
**Pull request by svenklemm**

### CODE REVIEW

**Review by erimatnor — APPROVED** [review]
Looks OK, but don't we have other SQL code that code benefit from pg_format?

**Comment by svenklemm** [comment]
> Looks OK, but don't we have other SQL code that code benefit from pg_format?

I think eventually we want to run it on all/most sql files but that probably requires some tuning steps to find suitable configuration, with my current config it splits function definitions weirdly

File: sql/pre_install/tables.sql [hunk 1]
```diff
@@ -40,302 +38,310 @@
...
-    ),
-    CHECK (
-        (integer_now_func_schema IS NULL AND integer_now_func IS NULL) OR
-        (integer_now_func_schema IS NOT NULL AND integer_now_func IS NOT NULL)
-    ),
-    UNIQUE (hypertable_id, column_name)
+  id serial NOT NULL PRIMARY KEY,
+  hypertable_id integer NOT NULL REFERENCES _timescaledb_catalog.hypertable (id) ON DELETE CASCADE,
+  column_name name NOT NULL,
+  column_type REGTYPE NOT NULL,
```
Comment (erimatnor) [inline sql/pre_install/tables.sql:90]: Is it intentional that pg_format leaves this type capitalized?

**Review by WireBaron — APPROVED** [review]
Not sure I agree with all of the changes, but I do like that we do have a uniform style now.

File: sql/views.sql [hunk 2]
```diff
@@ -91,179 +106,190 @@ FROM _timescaledb_config.bgw_job j
...
-          and N.nspname = cagg.user_view_schema ) viewinfo, LATERAL
-        ( select pg_get_viewdef(C.oid) as viewdefinition
-          FROM pg_class C LEFT JOIN pg_namespace N on (N.oid = C.relnamespace)
-          where C.relkind = 'v' and C.relname = cagg.direct_view_name
-          and N.nspname = cagg.direct_view_schema ) directview
-  WHERE cagg.mat_hypertable_id = ht.id;
+CREATE OR REPLACE VIEW timescaledb_information.continuous_aggregates AS
+SELECT cagg.user_view_schema AS view_schema,
+  cagg.user_view_name AS view_name,
+  viewinfo.viewowner AS view_owner,
```
Comment (erimatnor) [inline sql/views.sql:112]: Non-formatting nit. I guess we used just `owner` for the `hypertables` and `data_nodes` views. Perhaps we should adopt the same here?

File: sql/views.sql [hunk 3]
```diff
@@ -91,179 +106,190 @@ FROM _timescaledb_config.bgw_job j
...
+CREATE OR REPLACE VIEW timescaledb_information.chunks AS
+SELECT hypertable_schema,
+  hypertable_name,
+  schema_name AS chunk_schema,
+  chunk_name,
+  primary_dimension,
+  primary_dimension_type,
+  range_start,
+  range_end,
+  integer_range_start AS range_start_integer,
```
Comment (erimatnor) [inline sql/views.sql:164]: Also nit unrelated to formatting. A bit strange to first alias this to `integer_range_start` to then re-alias here to `range_start_integer`

**Review by erimatnor — COMMENTED** [review]
Pointed out a couple of view inconsistencies, not necessary to fix now.

File: sql/pre_install/tables.sql (hunk 1 above)
Comment (erimatnor) [reply sql/pre_install/tables.sql:90]: manually fix then?
